modules/ROOT/pages/ldap-synchronizer.adoc

```diff
@@ -582,3 +550,113 @@ In the following cases:
 * when the server certificate is auto-signed (use of custom root certification) you might configure :
  ** the `public certificate` should be imported into the default java or custom trust Store.
  ** the endpoint authentication might be disabled `disable_endpoint_authentication=true`
+
+[[deploy-with-docker]]
+=== Deploy LDAP Synchronizer with Docker
```

Comment:
Looking at the deployed page, 
The docker installation part is not very visible: [](https://bonitasoft-bonita-doc-build_preview-pr-1610.surge.sh/bonita/2021.1/ldap-synchronizer)

Maybe the page organisation needs to be adapted? e.g.
```asciidoc
= LDAP synchronizer
== LDAP objects supported (???)
== What does the LDAP synchronizer do
( content of the Running the synchronizer section )
== Using the LDAP synchronizer Docker image
== Using the LDAP synchronizer from the Bonita bundle
(also explain how to run the ldap synch)
== Configuration files
(configuration files)
== Advanced configuration
=== Using the LDAP synchronizer in a non-default tenant
=== LDAPS(TLS) activation
```
In any case there was and there still is an issue with the taxonomy in that page
![Screenshot 2021-07-06 at 16 32 33](https://user-images.githubusercontent.com/2739234/124618091-c5722a00-de77-11eb-8e7d-19de2eb49457.png)